scrapy/linkextractors/lxmlhtml.py

```diff
@@ -2,16 +2,19 @@
 Link extractor based on lxml.html
 """
 import six
-from six.moves.urllib.parse import urljoin
+try:
+    from scrapy.utils.url import urljoin
+except ImportError as e:
```

Comment:
Hm, now I see that it may happen indeed, but I think it's better to move fallback to ``scrapy.utils.url`` (see comment below: https://github.com/scrapy/scrapy/pull/3319#discussion_r200805153)